ocs_ci/utility/utils.py

```diff
@@ -940,17 +941,15 @@ def get_ocs_build_number():
         str: build number for ocs operator version
 
     """
-    from ocs_ci.ocs.resources.catalog_source import CatalogSource
+    from ocs_ci.ocs.resources.csv import get_csvs_start_with_prefix
```

Comment:
Is there a reason we are importing here as opposed to at the top of the file with all other imports? If so please add a comment explaining why we are importing here.

Reply:
Sorry I forgot to put it in comments, there was circular dependency hence I imported it here. I will update the comment

Reply:
done

Reply:
done